src/main/java/com/mtab/spring/boot/service/BkStrServiceV1.java

```diff
@@ -0,0 +1,72 @@
+package com.mtab.spring.boot.service;
+
+import com.mtab.spring.boot.service.models.Book;
+import com.mtab.spring.boot.service.models.BookPurchase;
+import com.mtab.spring.boot.service.models.NewPrice;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class BkStrServiceV1 implements BkStrService{
+
+    private final Map<String, Book> books = new HashMap<>();
+    public final double centsToDollars = 0.01;
```

Comment:
Normally, you would divide by 100 for the conversion, because even this value may not be precisely representable in floating point.

Also, It should not be public. By convention, final constants are static, and named in all caps - for example, `CENTS_TO_DOLLARS`.